test/Interop/C/bounds-safety/import-bounds-attributed-function.swift

```diff
@@ -0,0 +1,82 @@
+// RUN: %target-swift-ide-test -Xcc -fexperimental-bounds-safety-attributes -print-module -module-to-print=BoundsAttributedFunction -I %S/Inputs -source-filename=x | %FileCheck %s --check-prefixes=CHECK,C-ONLY
```

Comment:
It's a bit hard to discern the intention of this test (and the other tests too) since they just seems to be enumerating a bunch of otherwise meaningless combinations of annotations and language features. Can you add a comment at the top of the file explaining that the intention here is to just make sure all of these various annotations are being tested, and that the compiler is just expected to not misbehave here?